tests/integration/__tests__/cache.js

```diff
@@ -227,6 +227,95 @@ describe("--cache option", () => {
       );
     });
 
+    it("re-formats after execution without write.", async () => {
+      await runPrettier(dir, ["--cache", "--cache-strategy", "metadata", "."]);
```

Comment:
Let's put this call in a function, so we don't need check two places when updating. 

Maybe even better to accept `times` as a argument.

Reply:
Or simply reuse an array to store cli arguments.

Reply:
@fisker I've added `getCliArguments` util function. What do you think about [d653fe1](https://github.com/prettier/prettier/pull/13016/commits/d653fe10a92390102ae97cb43b2efcbb5c3c9140) ?

Reply:
It's unreadable... I only ask to make it easier to maintain the second format arguments by putting them together.

Reply:
Like this?: [9406d54](https://github.com/prettier/prettier/pull/13016/commits/9406d54946d7e80c30b26b66965e7419258955ca)

Reply:
Yes.